apps/explorer/src/pages/transaction-result/TransactionView.tsx

```diff
@@ -310,6 +310,22 @@ export function TransactionView({
                             />
                         )}
 
+                        {transaction.checkpoint && (
+                            <section className="py-12">
+                                <TableHeader>Checkpoint Detail</TableHeader>
+                                <div className="pt-4">
+                                    <DescriptionItem title="Checkpoint Seq. Number">
+                                        <Text
+                                            variant="body/medium"
+                                            color="sui-dark"
+                                        >
+                                            {transaction.checkpoint}
+                                        </Text>
```

Comment:
should this be a link to the checkpoint details page or no?

Reply:
ya good call, I have moved that to a link now